src/screens/Home/HomeScreen.js

```diff
@@ -2,15 +2,19 @@
 /* eslint-disable react/jsx-one-expression-per-line */
 /* eslint-disable react/prop-types */
 import React, { useState, useEffect } from 'react';
-import { StyleSheet, View, Text } from 'react-native';
+import {
+  StyleSheet, View, Text, Alert, TextInput,
+} from 'react-native';
 import { Button } from 'react-native-elements';
+import Firestore, { firebase } from '@react-native-firebase/firestore';
 import Auth from '@react-native-firebase/auth';
 import PropTypes from 'prop-types';
 import UploadFile from '../../components/UploadFile';
+import { INITIAL_USER_STATE } from '../../components';
```

Comment:
What is this INITIAL_USER_STATE??

Reply:
The initial userData of a user prior to account creation. This might be redundant here but I will double check.

Reply:
Double-checked - it isn't technically necessary, but it is probably a good precaution.